ews_calendar_sync.py

```diff
@@ -17,15 +17,19 @@
 
 import datetime
 import logging
-
+import argparse
 import toml
 from exchangelib import Credentials, Configuration, Account, DELEGATE, IMPERSONATION, CalendarItem
 import caldav
 import icalendar
 
 logger = logging.getLogger(__name__)
 
-with open('config.toml', encoding='utf-8') as f:
+parser = argparse.ArgumentParser(description='Read a TOML configuration file.')
```

Comment:
The `description` parameter of the ArgumentParser class is meant to describe the overall function of the script. Thus, it should be more like "Incrementally synchronizes a Microsoft Exchange calendar to a CalDAV server" or similar.

Reply:
You're right. My Python is a bit rusty but I found your script useful. Just needed to sync multiple accounts at the same time and though I'd share back.